<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>313</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="MplCanvas" name="tab">
           <attribute name="title">
            <string>Image 1</string>
           </attribute>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>110</y>
              <width>461</width>
              <height>81</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Open an image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
          <widget class="MplCanvas" name="tab_2">
           <attribute name="title">
            <string>Image 2</string>
           </attribute>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>110</y>
              <width>461</width>
              <height>81</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Open an image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
          <widget class="MplCanvas" name="tab_3">
           <attribute name="title">
            <string>Image 3</string>
           </attribute>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>110</y>
              <width>461</width>
              <height>81</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Open an image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="MplCanvas" name="widget_precessed" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>313</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>461</width>
             <height>81</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Processed image</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <widget class="QPushButton" name="pushButton_1">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>50</y>
             <width>191</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Trapezoidal</string>
           </property>
          </widget>
          <widget class="Line" name="line">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>0</y>
             <width>20</width>
             <height>331</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
          <widget class="Line" name="line_2">
           <property name="geometry">
            <rect>
             <x>630</x>
             <y>0</y>
             <width>20</width>
             <height>331</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>311</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Part 1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>10</y>
             <width>311</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Part 2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>640</x>
             <y>10</y>
             <width>311</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Part 3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>110</y>
             <width>191</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Wavy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>170</y>
             <width>191</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Circular</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>230</y>
             <width>191</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_5">
           <property name="geometry">
            <rect>
             <x>380</x>
             <y>50</y>
             <width>191</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Fuse</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_6">
           <property name="geometry">
            <rect>
             <x>700</x>
             <y>50</y>
             <width>191</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Hough</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplCanvas</class>
   <extends>QWidget</extends>
   <header>mplcanvas</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
